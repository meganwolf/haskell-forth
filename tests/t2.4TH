25 10 * 50 + .
25 10 -  CR .
5 2 / CR .
5 2 ^ CR .
5 3 ^ 8 + CR .
