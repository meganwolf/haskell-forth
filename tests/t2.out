300
15
2
25
133